Common/API/Prices.cs

```diff
@@ -0,0 +1,59 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using QuantConnect.Api;
+using QuantConnect.Util;
+
+namespace QuantConnect.API
+{
+    /// <summary>
+    /// Prices rest response wrapper
+    /// </summary>
+    public class Prices : RestResponse
```

Comment:
will we get `success` in each `prices` element?

Reply:
no 😖. It actually doesn't make sense for `Prices` to inherit from `RestResponse` anymore, I'll remove it.